.golangci.yml

```diff
@@ -122,30 +124,27 @@ linters:
     - staticcheck
     - structcheck
     - ineffassign
+    - megacheck
```

Comment:
This is not needed. It's just a combination of staticcheck, gosimple and unused. Can you remove it and a comment on top _not_ to add it back, since it's redundant.

Reply:
It's actually not redundant. megacheck reported many things that the other linters didn't.